### ISSUE TITLE
"usearch_sqlite" binary for Windows

### ISSUE THREAD

**Issue by fedehann** [unum-cloud/usearch#427]
### Describe what you are looking for

I tried to run the sample script to test out the USearch-SQLite3 extension:

```
import usearch
import sqlite3

conn = sqlite3.connect("data.db")
conn.enable_load_extension(True)
conn.load_extension(usearch.sqlite_path())
```

I got:

`Download failed: https://github.com/unum-cloud/usearch/releases/download/v2.12.0/usearch_sqlite_windows_amd64_2.12.0.dll could not be found`

I'm running Windows 10 and Python 3.12.3.

When I checked the releases at <https://github.com/unum-cloud/usearch/releases/>, I noticed that there are precompiled binaries of the USearch SQLite Extensions available for Mac and Linux, but not for Windows.

Would it possible to have precompiled binaries for Windows as well?

### Can you contribute to the implementation?

- [ ] I can contribute

### Is your feature request specific to a certain interface?

It applies to everything

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by ashvardanian** [comment]
Indeed, we only currently ship MacOS and Linux binaries. Compiling SQLite extensions for Windows was very challenging. If someone is ready to contribute - happy to merge 🤗
